collection/collection/src/commonTest/kotlin/androidx/collection/LruCacheTest.kt

```diff
@@ -1,553 +1,553 @@
-        assertEquals(
-            listOf("c" to "d", "a" to "b"),
-            snapshot1.entries.map { it.key to it.value },
-        )
-        // trigger access to move C to the end.
-        assertEquals("d", myCache["c"])
-        val snapshot2 = myCache.snapshot()
-        assertEquals(
-            listOf("a" to "b", "c" to "d"),
-            snapshot2.entries.map { it.key to it.value },
-        )
-        // Make sure it is mutable. This assertion doesn't make sense right now since the API
-        // returns MutableMap but we track APIs only w/ their Java signatures, which means a change
-        // to Map wouldn't show up in API files. Hence we have a test that would break if it starts
-        // returning immutable map.
-        snapshot1["c"] = "e"
-        assertEquals(
-            "e",
-            snapshot1["c"]
-        )
-    }
-
-    private fun newCreatingCache(): LruCache<String, String> =
-        object : LruCache<String, String>(3) {
-            override fun create(key: String): String? =
-                if (key.length > 1) "created-$key" else null
-        }
-
-    private fun newRemovalLogCache(log: MutableList<String>): LruCache<String, String> =
-        object : LruCache<String, String>(3) {
-            override fun entryRemoved(
-                evicted: Boolean,
-                key: String,
-                oldValue: String,
-                newValue: String?
-            ) {
-                log += if (evicted) "$key=$oldValue" else "$key=$oldValue>$newValue"
-            }
-        }
-
-    private fun assertHit(cache: LruCache<String, String>, key: String, value: String) {
-        assertEquals(value, cache[key])
-        expectedHitCount++
-        assertStatistics(cache)
-    }
-
-    private fun assertMiss(cache: LruCache<String, String>, key: String) {
-        assertEquals(null, cache[key])
-        expectedMissCount++
-        assertStatistics(cache)
-    }
-
-    private fun assertCreated(cache: LruCache<String, String>, key: String, value: String) {
-        assertEquals(value, cache[key])
-        expectedMissCount++
-        expectedCreateCount++
-        assertStatistics(cache)
-    }
-
-    private fun assertStatistics(cache: LruCache<*, *>) {
-        assertEquals(expectedCreateCount, cache.createCount(), "create count")
-        assertEquals(expectedPutCount, cache.putCount(), "put count")
-        assertEquals(expectedHitCount, cache.hitCount(), "hit count")
-        assertEquals(expectedMissCount, cache.missCount(), "miss count")
-        assertEquals(expectedEvictionCount, cache.evictionCount(), "eviction count")
-    }
-
-    private fun <T : Any> assertSnapshot(cache: LruCache<T, T>, vararg keysAndValues: T) {
-        val actualKeysAndValues = cache.snapshot().flatMap { (key, value) -> listOf(key, value) }
-        // assert using lists because order is important for LRUs
-        assertContentEquals(keysAndValues.asList(), actualKeysAndValues)
-    }
-}
+///*
```

Comment:
Commented out because it uses `runBlocking()` which is not available in Kotlin/JS.